spikes/api/main.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	mm "github.com/mattermost/platform/model"
+	"time"
+)
+
+/*
+main
+Usage: go run main.go -u <username> -p <password> <server-url> [team-name]
+Authenticates your login information, then gives you your AuthToken.
+If the team name is unentered or invalid main shows valid team names.
+*/
+func main() {
+	//Adds a  little clarity to the display
+	fmt.Println("---------------------------------------------------------")
```

Comment:
We should probably just make a function that does this. `printLine` or something.